### ISSUE TITLE
Add image to the template for most pages

### ISSUE THREAD

**Issue by mturilli** [radical-group/radical-group.github.io#34]
We should add an optional placeholder for an image to the following templates: 
- project list 
- project landing page
- News
- About

The placeholder should have predefined position and size. Once the template will be created, we will move to creating each single image. Future contributions should include an image.

**Comment by iparask** [comment]
I agree with that! Do you want to find an image and I will try to incorporate it to all templates? It will be similar to Darth Vaber for people!

**Comment by mturilli** [comment]
Previously, I found useful to work on size and positioning independent from an image. I created a png gray square in photoshop/gimp and scaled it down via HTML when trying different sizing and positioning.

About formatting, here a first spec for iteration:
- project list: rectangular (shorter hight) image on the left with the text on the right aligned top
- project landing page: header image below title and above description. Alternatively, rectangular image on the left with text wrap aligned top.
- News: rectangular image on the left with text wrap aligned top.
- About:
  - Research: rectangular image on the left with text wrap aligned top. Alternatively, short (as in hight) header image below the title and above the description. Note: the description should be probably shortened to a single paragraph.
  - Software development: header image below the title with architecture diagrams of the 4 RCT. Each image links to the respective github repo or the RCT web (but analytics is missing)

**Comment by mturilli** [comment]
See #59 for projects list and pages

**Comment by mturilli** [comment]
We need to add 'Results' to the project landing pages. I suggest to create a new entity `_result`, adding to the front matter metadata to connect the result to both a project (or more?) and to a publication (hopefully not more than one). The use of a dedicated entity will allow to control properties like caption, image, size of that image and so on, but also to embed the entity wherever we prefer like, say, an author personal page (just an example).

**Comment by mturilli** [comment]
Ping @iparask: what do you think about 'Results'? This is somehow blocking #33.

**Comment by iparask** [comment]
The results are going to be figures, right? I agree that adding it to the front matter is the best way to go. Especially if we want to quickly update them. I am not sure if we should include it in the publications. Since, the publication itself is a result of a project.

We could add related publications, similar to the personal pages.

**Comment by mturilli** [comment]
The idea is to have a new `_results` with entries that have only front matters. The idea is to be able to compose a figure with a caption and references to where it was published plus other metadata that, at the moment, does not come to mind.

**Comment by mturilli** [comment]
@iparask: should we converge on this? Maybe to be discussed in a brief dedicated meeting?
